src/pipecat/services/llm_service.py

```diff
@@ -42,8 +45,8 @@ async def __call__(
 
 
 @dataclass
-class FunctionCallEntry:
-    """Represents an internal entry for a function call.
+class FunctionCallRegistryItem:
+    """Represents an entry of our function call registry.
```

Comment:
This is one function call instance in the registry, right?

```suggestion
    """Represents an entry in our function call registry.
```